docs/core/tutorials/libraries.md

```diff
@@ -132,8 +132,8 @@ Your project file could look like this:
 You'll notice three major changes here:
 
 1. The `TargetFramework` node has been replaced by `TargetFrameworks`, and three TFMs are expressed inside.
-2. There is an `<ItemGroup>` node for the `net40 ` target pulling in one .NET Framework references.
-3. There is an `<ItemGroup>` node for the `net45` target pulling in two .NET Framework references.
+1. There is an `<ItemGroup>` node for the `net40 ` target pulling in one .NET Framework references.
+1. There is an `<ItemGroup>` node for the `net45` target pulling in two .NET Framework references.
 
 The build system is aware of the following preprocessor symbols used in `#if` directives:
```

Comment:
perhaps something to address on a different PR, but the list below should probably have .NET Framework 4.7 and .NET Standard 2.0. (haven't tested though)